### PR TITLE
[Twig] Document the new methods of AppVariable

### LINKED ISSUE #17265: [TwigBridge] Expose current route in `AppVariable`
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#47535
| PR author(s) | @HeahDude
| Merged in    | 6.2

### PR DESCRIPTION
**Pull request by javiereguiluz**
Fixes #17265.

### CODE REVIEW

File: templates.rst [hunk 1]
```diff
@@ -377,6 +377,17 @@ gives you access to these variables:
 ``app.token``
     A :class:`Symfony\\Component\\Security\\Core\\Authentication\\Token\\TokenInterface`
     object representing the security token.
+``app.current_route_name``
```
Comment (HeahDude) [inline templates.rst]:
```suggestion
``app.current_route``
```

File: templates.rst [hunk 2]
```diff
@@ -377,6 +377,17 @@ gives you access to these variables:
...
+    The name of the route associated to the current request or ``null`` if no
+    request is available (equivalent to ``app.request.attributes.get('_route')``)
+``app.current_route_parameters``
+    An array with the parameters passed to the route of the current request or an
+    empty array if no request is available (equivalent to ``app.request.attributes.get('_route_params')``)
+
+.. versionadded:: 6.2
+
+    The ``app.current_route_name`` and ``app.current_route_parameters`` variables
```
Comment (HeahDude) [inline templates.rst]:
```suggestion
    The ``app.current_route`` and ``app.current_route_parameters`` variables
```

**Review by HeahDude — CHANGES_REQUESTED** [review]
I was planning to do that this morning, thanks!

Also I thought we could update the last block of this section: https://symfony.com/doc/current/routing.html#getting-the-route-name-and-parameters.

**Comment by javiereguiluz** [comment]
Jules, thanks for the review and the fixes! (and thanks for implementing the feature too)

**Comment by xabbuh** [comment]
👍 I fixed a small typo in 3c35bf4abd1b9bad8e2fbe6762be0e6613312874

**Pull request merged**